edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/configsources/BackupConfigSourceTest.cs

```diff
@@ -165,7 +165,7 @@ public static DeploymentConfigInfo SetupNonEmptyDeployment(string moduleName = "
             var m1 = Mock.Of<IModule>();
             var modules = new Dictionary<string, IModule> { [moduleName] = m1 };
             SystemModules systemMods = new SystemModules(Option.None<IEdgeAgentModule>(), Option.None<IEdgeHubModule>());
-            return new DeploymentConfigInfo(1, new DeploymentConfig("1.0", runtime, systemMods, modules));
+            return new DeploymentConfigInfo(1, new DeploymentConfig("1.0", runtime, systemMods, modules, null));
```

Comment:
You may want to make a non-empty TwinIntegrity field here.  Does the integrity stored in backup get verified on reload?